jwala-services/src/main/java/com/cerner/jwala/service/resource/impl/ResourceServiceImpl.java

```diff
@@ -893,7 +900,10 @@ public CommandOutput generateAndDeployFile(final ResourceIdentifier resourceIden
             String resourceSourceCopy;
             final String deployFileName = resourceTemplateMetaData.getDeployFileName();
             final String deployPath = resourceTemplateMetaData.getDeployPath();
-            String resourceDestPath = deployPath + "/" + deployFileName;
+            resourceDestPath = deployPath + "/" + deployFileName;
+            if(resourceDestPath != null) {
```

Comment:
Would this ever be null? Even if the deployPath or deployFileName is null, they're being concatenated with a string constant ("/") so at the least it would "null/null" or if they're non-null empty strings "/".